Puzzlepart/did
# Showing friendly display message when an error occurs in Timesheet

### Your checklist for this pull request

### Review checklist
- [ ] Reload button. Do we want or should it be removed from PR?

### Description
Error handling in Timesheet. Showing friendly display message when an error occurs in Timesheet.

![image](https://user-images.githubusercontent.com/7606007/95055761-09bb7880-06f4-11eb-9612-d4970e559475.png)

Also re-added reload button. Do we want or should it be removed from PR?

![image](https://user-images.githubusercontent.com/7606007/95055897-3d969e00-06f4-11eb-9677-7d1e2fd4bfe2.png)


### How to test
1. Check out locally with [gh](https://github.com/cli/cli)
2. Add a throw to MSGraphService.getEvents to fake an error
3. Ensure there's an error message and not an empty list

### Related issues
Closes #567
